### PR TITLE
Various tweaks to posters.

### PR DESCRIPTION
**Pull request by spessbandit**
## About The Pull Request

- Posters now examine upon clicking them with help intent, rather than being torn down.
- Posters have a small art moodlet upon examining, depending on their type. The art moodlet is deliberately written to be non-poster-specific in case there are other things that may use it in the future.
- Fixes the time for the do_after on posters to actually track with the poster animation time.
- Adds one new corporate poster. 

wyci

## Why It's Good For The Game

fixes bugs and more fully fleshes out posters 

## Changelog
:cl: bandit
tweak: Clicking posters on help intent examines, not rips them. Posters now give you mood changes upon examining them. One new corporate poster is added. 
/:cl:

### CODE REVIEW

File: code/datums/components/art.dm [hunk 1]
```diff
@@ -49,3 +49,37 @@
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artgreat", /datum/mood_event/artgreat)
 	else
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artbad", /datum/mood_event/artbad)
+
+// NT/syndicate themed art, currently used in posters
+
+/datum/component/art/nt
+
+/datum/component/art/nt/on_obj_examine(datum/source, mob/M)
+	apply_moodlet(M, 0)
```
Comment (Qustinnus) [inline code/datums/components/art.dm]: Why are you overriding this proc

File: code/datums/components/art.dm [hunk 2]
```diff
@@ -49,3 +49,37 @@
...
+		postscript = "Glory to efficiency! Glory to potent plasma! Glory to Nanotrasen!"
+	else
+		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artbad", /datum/mood_event/artbad)
+		postscript = "Then you take some bodily fluids and apply them to the edge."
+	M.visible_message("<span class='notice'>[M] stops to inspect [parent].</span>", \
+						 "<span class='notice'>You take in the wholesome corporate message of \the [parent]. [postscript] </span>")
+
+/datum/component/art/syndicate
+
+/datum/component/art/syndicate/on_obj_examine(datum/source, mob/M)
```
Comment (Qustinnus) [inline code/datums/components/art.dm]: Ditto

File: code/datums/components/art.dm (hunk 1 above)
Comment (spessbandit) [reply code/datums/components/art.dm]: so it doesn't ask for impressiveness, see what the rev bust does

File: code/datums/components/art.dm [hunk 3]
```diff
@@ -49,3 +49,37 @@
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artgreat", /datum/mood_event/artgreat)
 	else
 		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artbad", /datum/mood_event/artbad)
+
+// NT/syndicate themed art, currently used in posters
+
+/datum/component/art/nt
```
Comment (ninjanomnom) [inline code/datums/components/art.dm]: You should not be making a subtype, work in any additional behavior into the base art component. In this case though this behavior doesn't look like it makes sense as part of this component at all and should be part of the poster code or a second component.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/datums/components/art.dm (hunk 3 above)
Comment (spessbandit) [reply code/datums/components/art.dm]: I'm just doing what the karl marx bust did

**Comment by actioninja** [comment]
Stale.

**Comment by spessbandit** [comment]
not stale, I am waiting for @ninjanomnom to reply to my comment above

**Comment by ninjanomnom** [comment]
There's nothing to respond to

**Comment by spessbandit** [comment]
yes there is:

a) why the fuck is it OK for the karl marx bust but not for this
b) could someone please explain to me how the inner workings of an addition to the game I made are going to have their functionality completely changed so I know how to use it

File: code/datums/components/art.dm (hunk 3 above)
Comment (ninjanomnom) [reply code/datums/components/art.dm]: I don't care what other code is doing, this is the wrong way to go about it

Comment (optimumtact) [reply code/datums/components/art.dm]: seems like a lot of effort for components that will be used once or twice in their life

Comment (ninjanomnom) [reply code/datums/components/art.dm]: Dealing with these changes in the proper way helps the code in the long run stay maintainable. Snowflake components subtypes really hurts the whole basis for their existence.

Comment (spessbandit) [reply code/datums/components/art.dm]: I don't have a preference as to how it's done although I would have to figure out how, I just don't realize why this was not brought up at that point

Comment (spessbandit) [reply code/datums/components/art.dm]: (also testing is going to be awkward until leg day is fixed)

**Pull request closed without merging**